cmd/lncli/profile.go

```diff
@@ -36,8 +37,16 @@ type profileEntry struct {
 
 // cert returns the profile's TLS certificate as a x509 certificate pool.
 func (e *profileEntry) cert() (*x509.CertPool, error) {
-	cp := x509.NewCertPool()
-	if !cp.AppendCertsFromPEM([]byte(e.TLSCert)) {
+	// Retrieve the system certificate pool. If not available (Windows),
+	// create a new pool.
+	cp, _ := x509.SystemCertPool()
```

Comment:
Apparently there's a workaround for the Windows problem: https://github.com/golang/go/issues/16736#issuecomment-540373689. Do you think it would be worth implementing that?
Also see my comment in #4549 about this.

Reply:
Maybe the question is: Do we need to provide a cert pool to `gRPC` if the cert is valid in the first place? Maybe it will fall back to the system CA store by default and we've been looking at this the wrong way all along?

Reply:
I tried with a new pool, but it doesn't work:
`[lncli] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: authentication handshake failed: x509: certificate signed by unknown authority"`

I don't know if the workaround is worth implementing. I think this PR is also valuable without it, so rather do it in a follow-up.

Reply:
Okay, I agree with not implementing the workaround in this PR.

I wasn't really clear about what I meant with "no cert pool". I meant, if instead of creating an empty pool or a pool with the system cert pool, what happens if you don't add the `grpc.WithTransportCredentials(creds)` at all? Does `gRPC` then fall back to the system pool? If so, we don't need `x509.SystemCertPool()`, as we don't really care about its content, just it being used.

Reply:
No, removing `grpc.WithTransportCredentials(creds)` doesn't work:

`[lncli] unable to connect to RPC server: grpc: no transport security set (use grpc.WithInsecure() explicitly or set credentials)`

Reply:
Apparently using `credentials.NewTLS(&tls.Config{})` instead of loading any certificates or pools causes `gRPC` to fall back to the system cert pool. I tested this with Windows as well and it seems to work.

Reply:
Great find. Updated